### DISCUSSION TITLE
Jenkins is down

### DISCUSSION THREAD

**Discussion by Chew** [eccentricdevotion/TARDIS#479] [General]
Cannot download. Kinda sad :(

**Comment by eccentricdevotion** [comment]
Should be back up now, computer dropped wifi while I was at work...

**Reply by Chew** [reply]
It happens, glad it's back! :D

**Comment by Chew** [comment]
Bad news

**Comment by eccentricdevotion** [comment]
Back now :)

**Comment by Chew** [comment]
It's down again :(

**Reply by eccentricdevotion** [reply]
Unfortunately I'm cycle touring in the French Alps right now, I'll see what I can do to get it back up, but some of the villages over here don't have wifi sometimes, and I'll have to talk my GF through it lol

**Reply by Chew** [reply]
All good, I've posted the latest JARs in the Discord so I hope that's useful 😄. Enjoy!

**Comment by Technoguyfication** [comment]
I'd be willing to host a Jenkins server for you on one of my servers, let me know if you'd like to get in touch. Free of charge, it's the least I can do to help support this project since I don't have time to make PRs anymore.

**Reply by Chew** [reply]
Would be nice to host a mirror just in case.

**Reply by eccentricdevotion** [reply]
@Technoguyfication Happy for you to host TARDIS mirror on your own Jenkins instance. If you do, let me know, and I'll add a mirror link to the appropriate web pages.
